src/ResourceManager/Compute/Commands.Compute/Generated/Disk/DiskGrantAccessMethod.cs

```diff
@@ -185,5 +185,8 @@ protected override void ProcessRecord()
             Mandatory = false)]
         [AllowNull]
         public int DurationInSecond { get; set; }
+
+        [Parameter(Mandatory = false, HelpMessage = "Run cmdlet in the background")]
+        public SwitchParameter AsJob { get; set; }
```

Comment:
All of these -AsJob parameters need covering tests: If the cmdlet does not currently have a covering test, please add one.  Otherwise, just covert the test as per: https://github.com/Azure/azure-powershell/wiki/PowerShell-Cmdlet-Design-Guidelines#asjob-parameter